projects/uitpas/reference/uitpas.json

```
@@ -2319,6 +2319,109 @@
       },
       "parameters": []
     },
+    "/passholders/me/checkin": {
```

```suggestion
    "/passholders/me/checkins": {
```

Vanuit het idee dat je 1 nieuwe checkin doet in een resource collectie van jouw "checkins"?